x/ccv/consumer/keeper/relay.go

```diff
@@ -36,6 +37,18 @@ func (k Keeper) OnRecvVSCPacket(ctx sdk.Context, packet channeltypes.Packet, new
 		k.SetProviderChannel(ctx, packet.DestinationChannel)
 		// - send pending slash requests in states
 		k.SendPendingSlashRequests(ctx)
+
+		// emit first VSC packet to signal that CCV is working
+		ctx.EventManager().EmitEvent(
+			sdk.NewEvent(
+				ccv.EventTypeFirstVSCPacket,
+				sdk.NewAttribute(sdk.AttributeKeyModule, types.ModuleName),
+				sdk.NewAttribute(channeltypes.AttributeKeySrcChannel, packet.SourceChannel),
+				sdk.NewAttribute(channeltypes.AttributeKeySrcPort, packet.SourcePort),
+				sdk.NewAttribute(channeltypes.AttributeKeyDstChannel, packet.DestinationChannel),
+				sdk.NewAttribute(channeltypes.AttributeKeyDstPort, packet.DestinationPort),
```

Comment:
I think we don't need all these fields. Only the channelID and portID on the consumer side.

Reply:
`Done`